src/main/java/net/mcreator/element/types/Particle.java

```diff
@@ -45,7 +46,6 @@ public class Particle extends GeneratableElement {
 
 	public double width;
 	public double height;
-	public double scale;

```

Comment:
I would keep ordering as is, applies to UI and test part of the code too

Reply:
ok